src/Evolution/DiscontinuousGalerkin/VariableFixing/RadiallyFallingFloor.hpp

```diff
@@ -0,0 +1,55 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include <algorithm>
+
+#include "DataStructures/DataVector.hpp"
+#include "DataStructures/Tensor/EagerMath/Magnitude.hpp"
+#include "DataStructures/Tensor/TypeAliases.hpp"
+#include "Domain/Tags.hpp"
+#include "Utilities/EqualWithinRoundoff.hpp"
+#include "Utilities/Gsl.hpp"
+#include "Utilities/TMPL.hpp"
+
+namespace VariableFixing {
```

Comment:
Why is this in `Evolution/DiscontinuousGalerkin`, is it related to DG?

Reply:
For now this is done so that it is in a location similar to where SlopeLimiters are, which are, as far as I'm aware, not specific to DG either. I'm open to placing it in a more appropriate location if we find we want to do so. (On further thought, EventsAndTriggers sounds like it could be an appropriate place for both of these things, at first glance. Considering SlopeLimiters are already in though I assume it was decided that DG was the best place for them, I would need someone else to chime in to be certain that this is indeed the right place for them.)

Reply:
@fmahebert believes that slope limiters are (at least primarily) used in DG.  src/Evolution/EventsAndTriggers is the infrastructure for events and triggers, not a place for arbitrary events to be placed (see, e.g., src/Time/Triggers) (and I don't think variable fixing is an event anyway).

This is hydro-specific, isn't it?  Do we have somewhere for hydro stuff?

Reply:
It is not DG specific.  I don't think we have anywhere yet for hydro stuff